package.json

```diff
@@ -47,6 +48,11 @@
   },
   "devDependencies": {
     "@babel/core": "^7.4.5",
```

Comment:
I don't see typescript in the deps, yet we have .ts files
and no build tasks in `scripts`
see https://github.com/senecajs/seneca-flow for an example of how a TS seneca plugin should be set up